### DISCUSSION TITLE
<Ctrl-`> doesn't work in vim/neovim

### DISCUSSION THREAD

**Discussion by GnikDroy** [microsoft/terminal#15750] [Q&A]
I can't bind <c-`> in Windows Terminal (Preview), with Neovim (nightly) even though things work fine in other terminal emulators (like kitty in Linux).

I have actually included the following in settings.json just to make sure.
```json
  "actions": [
    {
      "command": "unbound",
      "keys": "ctrl+`"
    }
  ],
```

I did actually search the issues and discussions for this, but couldn't find a similar issue.

**Comment by zadjii-msft** [comment]
This _might_ be #3483
